Vagrantfile

```diff
@@ -5,6 +5,11 @@
 # Exit if no environment name specified.
 # -------------------------------------------------------------
 
+$script = <<-SCRIPT
+docker cp /vagrant/.secrets.json web:/.secrets.json
+docker exec web ./events.py
```

Comment:
We should probably later convert these jsons to env files and then it's easier to run docker with `--env_file`
we should put it in the lists for the future

Reply:
agree. go ahead.